src/content/reference/react/createFactory.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: createFactory
+Judul : createFactory
 ---
 
 <Deprecated>
 
-This API will be removed in a future major version of React. [See the alternatives.](#alternatives)
+API ini akan dihapus pada versi utama React di masa depan [Lihat beberapa alternatifnya.](#alternatives)
```

Comment:
Do not translate major, it's related w semantic versioning term.

Or use `mayor` instead.

```suggestion
API ini akan dihapus pada versi mayor React berikutnya [Lihat beberapa alternatifnya.](#alternatives)
```